src/registrar/models/domain_application.py

```diff
@@ -477,6 +477,11 @@ def __str__(self):
         except Exception:
             return ""
 
+    def domain_is_not_active(self):
+        if self.approved_domain:
+            return not self.approved_domain.is_active()
```

Comment:
Wait please check me, but I believe this actually satisfies #854 

if you agree, please add your name to that ticket, remove zanders name, and update your PR  with this as being resolved as well

Reply:
EDIT: ignore the above, it doesn't fully meet 854. I added a comment onto your is active function as I wonder if there is a conflict here.

Reply:
Look at the tests, I think I've got this covered?